solutions/silver/ccc-Firehose.mdx

```diff
@@ -87,4 +88,147 @@ int main() {
 ```
 </CPPSection>
 
+<JavaSection>
+
+```java
+import java.io.*;
+import java.util.*;
+
+public class Firehose {
+	static final int MAX_H = 1000;
+	static final int STREET_SIZE = 1000000;
+
+	static boolean possible(int length, int numHouses, int numHydrants, int[] houses) {
```

Comment:
why have a `numHouses` argument if you can just have `houses.length`?

Reply:
done